lamindb_setup/dev/_settings_instance.py

```diff
@@ -44,6 +46,11 @@ def __repr__(self):
                 representation += f"\n- {attr}: {value}"
         return representation
 
+    @property
+    def id(self) -> Union[None, str]:
```

Comment:
I'd not expose it to the user for now, Bernardo. We might have a base62 id in the future, and then things will be confusing.

The instance UUID is a non-user facing internal thing.